src/term.rs

```diff
@@ -185,6 +185,12 @@ pub struct RecordAttrs {
     pub open: bool,
 }
 
+#[derive(Debug, Default, Eq, PartialEq, Clone)]
```

Comment:
This structure could have a short description, at least of the fields (`binding_type` and `rec`).